handel/src/services/ecs-fargate/params-schema.json

```diff
@@ -0,0 +1,300 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "type": "object",
+    "properties": {
+        "type": {
+            "type": "string",
+            "description": "The service type (should always be 'ecs-fargate')",
+            "pattern": "^ecs-fargate$",
+            "errorMessage": "The 'type' parameter must be 'ecs-fargate'"
+        },
+        "max_mb": {
+            "type": "integer",
+            "errorMessage": "The 'max_mb' parameter must be an intiger"
```

Comment:
Spelling error here and on a few other lines in the schema file: 'intiger' should be 'integer'.